assets/dev/js/editor/document/dynamic/commands/base/disable-enable.js

```diff
@@ -0,0 +1,59 @@
+import History from '../../../commands/base/history';
+import ElementsSettings from '../../../elements/commands/settings';
+
+export default class DisableEnable extends History {
+	static restore( historyItem, isRedo ) {
+		const data = historyItem.get( 'data' );
+
+		// Upon `disable` command toggle `isRedo`.
+		if ( 'document/dynamic/disable' === data.command ) {
+			isRedo = ! isRedo;
+		}
+
+		historyItem.get( 'containers' ).forEach( ( container ) => {
+			const settings = data.changes[ container.id ],
+				toggle = isRedo ? 'enable' : 'disable';
```

Comment:
Use full command - for easy find it in the the code.
```suggestion
				toggle = isRedo ? 'document/dynamic/enable' : 'document/dynamic/disable';
```